src/config/environmentConfig.ts

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+/**
+ * Validates and sets the environment configuration based on the provided schema.
+ * If the environment config is invalid, an error is thrown.
+ *
+ * @throws {Error} If the environment config is invalid
+ *
+ * @returns {EnvironmentConfig} The validated environment configuration
+ */
+
+import * as joi from '@hapi/joi';
+import { EnvironmentConfig } from '../models/environmentConfig.interface';
+
+const envs = ['local', 'development', 'qa', 'production', 'test'];
+
+if ([envs[0], envs[4]].includes(process.env.NODE_ENV!)) {
+  /* eslint-disable  @typescript-eslint/no-var-requires */
+  require('dotenv').config();
```

Comment:
why can't we use import. any specific reason to skip for other environments?

Reply:
Now we are using import. Regarding skipping the other environments, `local` is for our localhost and `test` is for jest execution. So we are loading the `.env` configuration for only these two environments. For other environments, all the configuration comes from Azure settings.